### PR TITLE
Edit SSPOperatorDown

### PR DESCRIPTION
**Pull request by apinnick**
Signed-off-by: Avital Pinnick <apinnick@redhat.com>

### CODE REVIEW

**Comment by apinnick** [comment]
@sradco Ready for review

File: docs/runbooks/SSPOperatorDown.md [hunk 1]
```diff
@@ -1,30 +1,43 @@
 # SSPOperatorDown
+<!--apinnick, Nov. 2022-->
 
 ## Meaning
 
-The SSP Operator is responsible for deploying and reconciling the common-templates and the template validator.
-This alert fires when the SSP operator is down.
+This alert fires when the Scheduling, Scale and Performance (SSP) Operator is down.
+
+The SSP Operator is responsible for deploying and reconciling the common templates and the Template Validator.
```
Comment (sradco) [inline docs/runbooks/SSPOperatorDown.md:8]: @apinnick I believe this line should be added also to this PR https://github.com/kubevirt/monitoring/pull/115/files.

**Comment by sradco** [comment]
@borod108 Please review this PR

File: docs/runbooks/SSPOperatorDown.md (hunk 1 above)
Comment (borod108) [reply docs/runbooks/SSPOperatorDown.md:8]: good idea

**Comment by borod108** [comment]
lgtm

**Pull request merged**